packages/backend/src/open_payments/wallet_address/service.ts

```diff
@@ -165,8 +176,9 @@ async function createWalletAddress(
 
 async function updateWalletAddress(
   deps: ServiceDependencies,
-  { id, status, publicName }: UpdateOptions
+  { id, status, publicName, additionalProperties }: UpdateOptions
 ): Promise<WalletAddress | WalletAddressError> {
+  const trx = await WalletAddress.startTransaction()
   try {
     const update: UpdateInput = { publicName }
     const walletAddress = await WalletAddress.query(deps.knex)
```

Comment:
not a big deal but should we also use the `trx` here?

Reply:
I suppose probably so... and more importantly the underlying `IncomingPayment.patch`in `deactivateOpenIncomingPaymentsByWalletAddress` (called a few lines below this).